modules/users/tests/server/user.server.model.tests.js

```diff
@@ -67,6 +67,16 @@ describe('User Model Unit Tests:', function() {
 				done();
 			});
 		});
+
+		it('should confirm that saving user password doesnt change the password', function(done) {
```

Comment:
Should this read 'should confirm that saving user *model* doesnt change the password'.  Also, should we have a test for change password now?  I know these tests need to be built out more, in general, but these are a good start.